pages/in/widget/iscn-ar/index.vue

```diff
@@ -516,6 +467,7 @@ export default {
       license,
       url,
     });
+    this.transactionFee = this.actualSendAmount === 0 || this.actualSendAmount === '0' ? this.ISCNTotalFee : this.amounts[0];
```

Comment:
I think we can refactor `this.actualSendAmount === 0 || this.actualSendAmount === '0` as a computed, and cast it into a number/boolean
it is very easy to reuse `actualSendAmount` wrongly in its raw form
When do we expect it to be a number and string?

Reply:
I think this.actualSendAmount is always a string, as it is coming from  .toFixed()

Reply:
Will rename `this.actualSendAmount === '0' `to `shouldSkipLIKEPay`